includes/Modules/Ads.php

```diff
@@ -85,7 +93,64 @@ protected function setup_assets() {
 					),
 				)
 			),
+			new Script_Data(
+				'googlesitekit-ads-pax-config',
+				array(
+					'global'        => '_googlesitekitPAXConfig',
+					'data_callback' => function() {
+						if ( ! current_user_can( Permissions::VIEW_DASHBOARD ) ) {
```

Comment:
`VIEW_DASHBOARD` is equivalent to being able to view _any_ version of the dashboard. We only want to include this data if the user can view the main dashboard which has a dedicated capability. Same below as well.